FAQ.md

```diff
@@ -0,0 +1,41 @@
+# FAQ
+
+This file contains Frequently Asked Questions and their answers. Please feel free to make PRs to amend this file with new questions.
+
+## Any audio/music/sounds related problems
+
+Flame only provides a thin wrapper over the [audioplayers](https://github.com/luanpotter/audioplayers) library. Please make extra sure the problem you are having is with Flame. If you have a low-level or hardware related audio problem, probably it's something related to audioplayers, so please feel free to head to that repository to search your problem or open your issue. If you indeed have a problem with the Flame wrap around audioplayers, please feel free to open an issue.
+
+## Drawing over the notch (for Android)
+
+In order to draw over the notch, you must add the following line to your `styles.xml` file:
+
+```xml
+<item name="android:windowLayoutInDisplayCutoutMode">shortEdges</item>
+```
+
+For more details, please check [this PR](https://github.com/impulse/flutters/commit/25d4ce726cd18e426483e605fe3668ec68b3c12c) from [impulse](https://github.com/impulse).
+
+## Examples
+
+We have a few examples! The simplest of all is in [the root example folder](/example/). This folder must be called `example` and contain a single project, so it's a very simple generic example game.
+
+However there is an [examples folder inside the docs](/doc/examples) that contains many more examples. Most newer features we are adding we are trying to add a specific example for that. Those are directed to specific features.
+
+If you want a more full-fledged game, please check [BGUG](https://github.com/fireslime/bgug), FireSlime's open source fast paced side-scrolling platformer. It is mostly up-to-date with Flame and uses a good chunck of the featues offered by the engine. Other examples game are [feroult](https://github.com/feroult)'s [Haunt](https://github.com/feroult/haunt), which is outdated but showcases Box2D, and also [renancaraujo](https://github.com/renancaraujo)'s port of the [trex](https://github.com/flame-engine/trex-flame) chrome game in Flame.
```

Comment:
What about bob?

Reply:
sorry, I forgot

Reply:
had I had flame awesome this would not have happened hahaha